resources/views/autodesk.blade.php

```diff
@@ -0,0 +1,18 @@
+@extends('layouts/app')
+
+    @section('title')
+    Autodesk Access Request | {{ config('app.name') }}
+@endsection
+
+    @section('content')
+    @component('layouts/title')
+    Autodesk Access Request
+    @endcomponent
+
+    <p>
+{{ $message }}
+</p>
+
+<a href="/">Go back to the dashboard</a>
+
+                               @endsection
```

Comment:
Checks don't enforce style in views. The indentation is weird here. The content is fine though.